x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/planner/QueryFolderOkTests.java

```diff
@@ -114,6 +110,11 @@ private static void addSpec(ArrayList<Object[]> arr, String name, String query,
     }
 
     public void test() {
+        // skip tests that do not make sense from case sensitivity point of view
```

Comment:
It's idiomatic to use `assume` which will indicate to the framework that the test has not actually ran.